src/drivers/simulink/simulink.py

```diff
@@ -0,0 +1,172 @@
+    '''
+
+    def __init__(self, name: str, pipe: Optional[Pipe] = None):
+        """
+        :param name: (optional) Name for the driver
+        :param pipe: (optional) Pipe used to communicate with the driver thread. See gateway.py
+        """
+        # Inherit
+        driver.__init__(self, name, pipe)
+
+        # Parameters
+        self.session_name = "any"
+        self.simulink_block = "simulink_session/block"
+        self.autorun_simulink = False
+
+        # Internal
+        self._runtime_object = None
+        self._rto_name = None
+
+
+    def connect(self) -> bool:
+        """ Connect driver.
+
+        : returns: True if connection stablished False if not
+        """
+        try:
+            if not MATLAB_SDK_FOUND:
+                add_matlab_path()
+                import matlab.engine
+            
+            # Get list of shared matlab sessions
+            sessions = matlab.engine.find_matlab()
+            assert sessions, "No shared matlab sessions found"
+            if self.session_name != "any":
+                # id specified, try connecting to that session
+                self._connection = matlab.engine.connect_matlab(self.session_name)
+            else:
+                self._connection = matlab.engine.connect_matlab(sessions[0])
+
+            # Start Simulink if autorun is enabled
+            if self.autorun_simulink:
+                self._connection.eval("set_param('simulink_test1','SimulationCommand','Start');", nargout=0)
+            
+            # Create temporary RuntimeObject for read/write operations
+            self._rto_name = 'rto_' + self.simulink_block.replace(" ", "_").replace("/","_")
+            self._runtime_object = self._connection.eval(f"get_param('{self.simulink_block}', 'RuntimeObject');")
+            self._connection.workspace[self._rto_name] = self._runtime_object
+
+            return True
+
+        except Exception as e:
+            self.sendDebugInfo('SETUP failed: ' + str(e))
+
+        return False
+
+
+    def disconnect(self):
+        """ Disconnect driver. Remove the temporary rto variable. """
+        self._connection.eval('clear ' + self._rto_name, nargout=0)
+        if self.autorun_simulink:
+            self._connection.eval("set_param('simulink_test1','SimulationCommand','Stop');", nargout=0)
+        self._connection.exit()
+
+
+    def addVariables(self, variables: dict):
+        """ Add variables to the driver. Correctly added variables will be added to internal dictionary 'variables'.
+        Any error adding a variable should be communicated to the server using sendDebugInfo() method.
+        : param variables: Variables to add in a dict following the setup format. (See documentation)
+
+        """
+        self.variables.update(variables)
+        for var_id in self.variables:
+            self.variables[var_id]['value'] = self.defaultVariableValue(self.variables[var_id]['datatype'], self.variables[var_id]['size'])
```

Comment:
I think we should check here whether the variable is actually found or not in Matlab. If it is not found, it should not be added to self.variable and the user should be warned by using self.sendDebugVarInfo().